_posts/august-2024-newsletter.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "August Newsletter"
+excerpt: "Our August newsletter highlights key events, including our General Meeting on August 18 and upcoming public meetings at the DART Board and Dallas City Council."
+coverImage: "/assets/blog/august-2024-newsletter/cover.jpg"
+date: "2024-08-09T05:00:00.000Z"
+author:
+  name: DATA Newsletter
+  picture: "/assets/blog/authors/data-team.png"
+ogImage:
+  url: "/assets/blog/august-2024-newsletter/cover.jpg"
+---
+
+We have three upcoming events that I want you to be aware of this month.  
+
+## August General Meeting
+
+---
+First, our August General Meeting will be on August 18, 2024 from 3pm to 5pm at the Metropolitan [1200 Main Street, Dallas](https://maps.app.goo.gl/pPBSUsQeJYZr1hxf7) (Enter via Murphy Drive) on the 6th floor, same room as the first meeting we held. We will have light snacks and refreshments available.
+
+We will be discussing our strategy for two upcoming public meetings that we want to show up in large numbers for.
+
+## DART Board Meeting
+
+---
+
+The first meeting is the DART Board meeting at [DART Headquarters](https://maps.app.goo.gl/36uCk3suvRdJPfM47) on August 27, 2024 at 6pm, please arrive 30 minutes early to register for public comments.
+
+Some members of the DART board are not very supportive of transit. Right now, there is a push among some board members to cut DART’s operating budget by 5-10% this year, and cap DART’s revenue growth. Other DART board members have expressed willingness to compromise with these cities that want to cut DART’s funding.
+
+We are here to to communicate 4 things
+
+1. Our opposition to these draconian budget cut resolutions, and our disappointment with the member cities who passed them
+2. More frequency/service improvements in next year's budget.
+3. Our opposition to ANY revenue capping.
+4. Express how DART should oppose any state legislation that cuts/caps their budget
+
+## Dallas City Council Meeting
+
+---
+The second meeting is the City of Dallas City Council meeting at [City Hall](https://maps.app.goo.gl/BgTdpbSjKRe73pF88) on August 28, 2024 at 9am. We intend to make a big showing here in support of DART funding, and communicate that we are:
+
+1. ***Opposed*** to DART Cuts
+2. ***Opposed*** to DART revenue caps
+3. ***Opposed*** to any state legislation that seeks to cap/cut DART’s revenue. We do NOT need to get the state legislature involved in DART’s budget.
+
+This meeting is just one day after the DART board meeting. If you can only attend one meeting, Dallas residents should prioritize this one.
+
+Registration for public comments will close at 5pm on August 27th.If you plan on attending this meeting in-person, please let us know so we can get a count. The City of Dallas also allows speaking virtually as an option in the event you are unable to attend due to other obligations.
```

Comment:
```suggestion
Registration for public comments will close at 5pm on August 27th. If you plan to attend this meeting in-person, please let us know so we can get a count. The City of Dallas also allows speaking virtually as an option in case you cannot attend in-person.
```